concordium-consensus/src/Concordium/GlobalState/Persistent/Account/StructureV1.hs

```diff
@@ -1019,6 +1022,8 @@ data PersistentAccount av = PersistentAccount
       -- | The actively staked balance of the account.
       --  INVARIANT: This is 0 if the account is not a baker or delegator.
       accountStakedAmount :: !Amount,
+      -- | The protocol level tokens of the account in ascending order of the TokenIndex.
+      accountTokens :: !(Conditionally (SupportsPLT av) TokenAccountStateTable),
```

Comment:
I think the `TokenAccountStateTable` should be stored under a reference so that it doesn't need to be stored multiple times if it doesn't change. It also needs to be hashed and included in the account hashing where relevant.

Reply:
Done.